eng/scripts/update-packagejson-links.js

```diff
@@ -0,0 +1,12 @@
+const path = require("path");
+const fs = require("fs");
+
+const args = process.argv.slice(2);
+const packageJson = args[0];
+const packageVersion = args[1];
+
+const fileContent = fs.readFileSync(packageJson);
+const updatedContent = fileContent.toString().replace(/\"link:.*\"/g, `">=${packageVersion}"`);
+if (fileContent != updatedContent) {
+    fs.writeFileSync(packageJson, updatedContent);
```

Comment:
👏  much better than trying to do this with MSBuild.

Reply:
My only concern was that we now invoke `node` in MSBuild. But presumably we already do that as part of `yarn`